ui/dashboards/src/context/DashboardProvider/common.ts

```diff
@@ -30,3 +32,37 @@ export function generateId() {
   }
   return globalThis.dashboardStoreId++;
 }
+
+// Helper function to create initial PanelDefinitions
+export function createPanelDefinition(
+  defaultPanelKind?: string,
+  defaultPanelSpec?: UnknownSpec,
+  defaultTimeSeriesQueryKind?: string
+): PanelDefinition {
+  return {
+    kind: 'Panel',
+    spec: {
+      display: {
+        name: '',
+        description: undefined,
+      },
+      plugin: {
+        kind: defaultPanelKind ?? '',
+        spec: defaultPanelSpec ?? {},
+      },
+      queries: [
```

Comment:
This is another bit that could get interesting if we ever support not-time-series queries.

Reply:
i updated this so `queries` can be an empty array